Sources/Core/Network/Client+Error.swift

```diff
@@ -64,8 +64,9 @@ public extension Client {
 	///
 	/// - returns: An error with the message from the response (see `ClientError`), or the existing error
 	static func extract<T>(from response: DataResponse<T, AFError>, error: AFError) -> Error {
-		// unauthorized status code --> unauthorized
-		if let status = response.response?.statusCode, status == 401 || status == 403 {
+		if case .explicitlyCancelled = error {
```

Comment:
Doesn't this need wrapping in "available iOS 13"?

Reply:
This is an Alamofire error and has has nothing to do with async.